src/plugins/scriptedwizard/resources/wxwidgets/wizard.script

```diff
@@ -747,24 +741,26 @@ function SetupProject(project)
             project.AddLinkLib(LibPrefix + _T("oleaut32") + LibSuffix);
             project.AddLinkLib(LibPrefix + _T("uuid") + LibSuffix);
             project.AddLinkLib(LibPrefix + _T("comctl32") + LibSuffix);
-            project.AddLinkLib(LibPrefix + _T("wsock32") + LibSuffix);
-            project.AddLinkLib(LibPrefix + _T("odbc32") + LibSuffix);
+            if (LibWxLinkWinSock2)
```

Comment:
Why do you want this to be an option?

Reply:
wxWidgets can be built using either Winsock  or Winsock2, depending on the value of `wxUSE_WINSOCK2`  define in setup.h. This value defaulted to `0` up to wxWidgets 3.1.5 but is set to `1` since 3.1.6, see https://github.com/wxWidgets/wxWidgets/commit/86d1a25624b0a3d34005a57bd299061560cc933c

Hence, in a project using the static wxWidgets build the matching winsock library must be linked with. AFAIK the project wizard is unable to obtain define values from the build-specific setup.h, so this needs to be set accordingly by the user. I planned on presetting the checkbox to true for wxWidgets 3.2+.

Reply:
OK, Interesting, If I'm an unsuspecting user would I be able to set this correctly only by the description in the UI?

Reply:
I could not figure out a better solution. I believe the one I have is still better than nothing and can avoid linking errors. 

If you can think of something better, I will gladly hear it.